### ISSUE TITLE
Add support for cephx access type

### ISSUE THREAD

**Issue by mtneug** [terraform-providers/terraform-provider-openstack#714]
The `openstack_sharedfilesystem_share_access_v2` resource is currently missing the `cephx` access type and something like `access_key` as output.

### Terraform Version

```
Terraform v0.11.13
+ provider.openstack v1.16.0
```

### Affected Resource(s)

* `openstack_sharedfilesystem_share_access_v2`

### Terraform Configuration Files

```hcl
resource "openstack_sharedfilesystem_share_access_v2" "example" {
  share_id     = "${openstack_sharedfilesystem_share_v2.share.id}"
  access_type  = "cephx"
  access_level = "rw"
  access_to    = "myuser"
}
```

**Comment by jtopjian** [comment]
@mtneug Thanks!

I have opened #715 to make these changes. However, I don't have access to a Manila environment in order to verify them. Would you be able to test this?

**Comment by mtneug** [comment]
Wow, that was quick.

Yes, I can test it. See comment in your PR.

**Comment by jtopjian** [comment]
Implemented in #715
